test/mobile/test_lite_script_type.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch.utils.bundled_inputs
+import io
+from typing import NamedTuple
+
+from torch.jit.mobile import _load_for_lite_interpreter
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+
+class TestLiteScriptModule(TestCase):
+
+    def test_namedtuple_custom_classtype(self):
+        class Foo(NamedTuple):
+            id: torch.Tensor
+
+        class Bar(torch.nn.Module):
+            def __init__(self):
+                super(Bar, self).__init__()
+                self.foo = Foo(torch.tensor(1))
```

Comment:
It's great to have Class of NamedTuple supported in different scenarios: creation, return, attribute access. Would be nice to add those supported scenarios in the comments. 

Could you also verify if it has Function access?

Reply:
Got it. So inlining actually helps. Maybe it's something to keep in mind that when we support CALL and not everything is inlined, we may revisit this issue.